src/passes/embedwast.cpp

```diff
@@ -0,0 +1,24 @@
+#include <string>
```

Comment:
How about if we make this a python script instead? (I actually see we have `scripts/process_optimize_instructions.py` which is sort of similar, too.)

Reply:
Sure thing